src/main/java/eu/bbmri_eric/negotiator/dto/discoveryservice/GenericOrganization.java

```diff
@@ -0,0 +1,7 @@
+package eu.bbmri_eric.negotiator.dto.discoveryservice;
+
+public interface GenericOrganization {
```

Comment:
missing javadoc

Reply:
Fixed in 3235f439a3ce3690a16b4781597181da46cebe37